### DISCUSSION TITLE
PieChart with needle?

### DISCUSSION THREAD

**Discussion by J4v4Scr1pt** [plouc/nivo#2620] [Q&A] [answered]
Maybe I have missed something, but is there a chart simillar to this?
If not, is it possible to do with this library?
![image](https://github.com/plouc/nivo/assets/5929055/aa1e1fa7-146b-42ad-94ad-21110e2c7694)

**Comment by plouc** [comment]
It's not possible by default, but you could use a custom layer for this.

**Reply by plouc** [reply]
Assuming you're comfortable with SVG.

**Comment by J4v4Scr1pt** [comment]
Thanks for the ultra super quick response 🚀 !

I would say not to comfortable with SVG. 😅 

Is there an example of a layer anywhere in the docs?

**Reply by plouc** [reply, marked as answer]
There's a [story involving a custom layer](https://nivo.rocks/storybook/?path=/story/pie--adding-a-metric-in-the-center-using-a-custom-layer), although it's much simpler as it's just a text, you can find the source [here](https://github.com/plouc/nivo/blob/master/storybook/stories/pie/Pie.stories.tsx#L147).

**Reply by J4v4Scr1pt** [reply]
Thanks man!
